core-models/src/main/java/com/anytypeio/anytype/core_models/ObjectWrapper.kt

```diff
@@ -239,11 +240,19 @@ sealed class ObjectWrapper {
         val color: String = relationOptionColor.orEmpty()
     }
 
-    data class Workspace(override val map: Struct) : ObjectWrapper() {
+    data class SpaceView(override val map: Struct) : ObjectWrapper() {
         private val default = map.withDefault { null }
         val id: Id by default
         val name: String? by default
-        val spaceId: String? by default
+        val targetSpaceId: String? by default
```

Comment:
what's targetSpaceId for Space?

Reply:
`targetSpaceId` is the `ID` of the space object corresponding to this `SpaceView` (visual representation of this space object, having different `ID`)